api/_db.js

```diff
@@ -89,10 +93,14 @@ const fetchOne = async (client, id) => {
 	const filterColumnIdx = headers.indexOf(filterColumnHeader);
 	const certificateColumnIdx = headers.indexOf(certificateHeader);
 	const nameColumnIdx = headers.indexOf(nameHeader);
+	const examinerColumnIdx = headers.indexOf(examinerHeader);
+	const examTimeColumnIdx = headers.indexOf(examTimeHeader);
 
 	assert(~filterColumnIdx, `Filter column not found. Looked for ${filterColumnHeader}`);
 	assert(~certificateColumnIdx, `Certificate column not found. Looked for ${certificateHeader}`);
 	assert(~nameColumnIdx, `Certificate column not found. Looked for ${nameHeader}`);
+	assert(~examinerColumnIdx, `Certificate column not found. Looked for ${examinerHeader}`);
```

Comment:
The error message should also be rewritten. It's not the "Certificate column" anymore, but the column for examiner, etc.